accepted/00097-recurring-actions.md

```diff
@@ -0,0 +1,321 @@
+- Feature Name: Recurring Actions
+- Start Date: 2023-03-01
+
+# Summary
+[summary]: #summary
+
+Extend the recurring highstate framework to support other recurring action types.
+The goal here is to have a framework that supports the implementation of currently planed action types as well as new ones in the future.
+
+# Motivation
+[motivation]: #motivation
+
+- Why are we doing this?
+Customers want to be able to further automate their workflows. Recurring highstate was one step in this direction but there are some areas that would greatly benefit from being automated.
+
+- What use cases does it support?
+Schedule recurrent execution of administration tasks that used to need manual scheduling in the past.
+
+- What is the expected outcome?
+Have a framework that allows the creation of recurrent action schedules of currently planed as well as potential future actions.
+
+# Detailed design
+[design]: #detailed-design
+
+## General design
+
+The existing recurring highstate framework already provides us with the bulk part of what be need for a generalized implementation. Some changes to the `RecurringAction` entity are required to make it work with multiple recurring action types. See details below.
+
+### Database
+
+**Solution #1:**
```

Comment:
Is it already clarified which "Solution" you want to use? If yes, it is time to remove the other one which is out of the game. A final RFC should only describe the future implementation and provide no room for interpretations (if possible)